x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/ElasticInferenceServiceSparseEmbeddingsModel.java

```diff
@@ -105,9 +105,9 @@ private URI createUri() throws URISyntaxException {
 
         switch (modelId) {
             case ElserModels.ELSER_V2_MODEL -> modelIdUriPath = "ELSERv2";
-            default -> throw new IllegalArgumentException("Unsupported model for EIS [" + modelId + "]");
+            default -> throw new IllegalArgumentException("Unsupported model for Elastic Inference Service [" + modelId + "]");
```

Comment:
Maybe we could use a constant for `Elastic Inference Service` to make sure it stays consistent throughout log messages, error messages etc. I think we already have two places (here and in the deprecation log warning message), right?

We could add it to `ElasticInferenceService` and maybe name it `HUMAN_READABLE_NAME` or something like that to distinguish from `NAME`?

Reply:
Good idea! Where should that live in your opinion?

Reply:
Thanks! 

I think I would prefer if "human readable name" constant lives alongside the service provider `NAME` inside [ElasticInferenceService.java](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/elastic/ElasticInferenceService.java). WDYT?